ci/scripts/image_scripts/build_ipa_on_centos.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Set execution parameters to:
+# Fail whenever any command fails
+set -eu
+
+# Configurable options
+DISABLE_UPLOAD="${DISABLE_UPLOAD:-false}"
+IPA_BUILDER_BRANCH="${IPA_BUILDER_BRANCH:-master}"
+RT_UTILS="${RT_UTILS:-/tmp/utils.sh}"
+IPA_BUILDER_PATH="${IPA_BUILDER_PATH:-ironic-python-agent-builder}"
+IPA_IMAGE_NAME="${IPA_IMAGE_NAME:-ironic-python-agent}"
+IPA_IMAGE_TAR="${IPA_IMAGE_NAME}.tar"
+IPA_BUILD_WORKSPACE="${IPA_BUILD_WORKSPACE:-/tmp/dib}"
+IPA_REPO="${IPA_REPO:-https://github.com/Nordix/ironic-python-agent}"
+IPA_REPO_CONF="${IPA_IMAGE_NAME} git /tmp/ironic-python-agent ${IPA_REPO}"
+IPA_REPO_CONF_PATH="${IPA_BUILDER_PATH}/dib/ironic-python-agent-ramdisk/source-repository-ironic-python-agent"
+RT_URL="${RT_URL:-https://artifactory.nordix.org/artifactory}"
+# Install required packages
+sudo apt install --yes python3-pip  python3-virtualenv qemu-utils
+
+# Create the work directory
+mkdir "${IPA_BUILD_WORKSPACE}"
```

Comment:
```suggestion
mkdir -p "${IPA_BUILD_WORKSPACE}"
```
In case we change the directory to be non-existing subdirectory